features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -725,6 +725,9 @@ internal open class RumViewScope(
         val memoryInfo = lastMemoryInfo
         val refreshRateInfo = lastFrameRateInfo
         val isSlowRendered = resolveRefreshRateInfo(refreshRateInfo) ?: false
+        // make a copy - by the time we iterate over it on another thread, it may already be changed
+        val eventFeatureFlags = featureFlags.toMutableMap()
+        val eventAdditionalAttributes = attributes.toMutableMap()
```

Comment:
this one should be used below as here https://github.com/DataDog/dd-sdk-android/pull/1830/files#diff-ab30afe3da5a0243ae755135af0e1296b0137b5794ad63b8ec03d9731a5d75d2R827